core/src/test/java/com/linecorp/armeria/server/ServiceRequestContextTest.java

```diff
@@ -67,6 +72,11 @@ void currentOrNull() {
         try (SafeCloseable unused = ctx.push()) {
             assertThat(onEnterExitStack).hasSize(1);
             assertThat(ServiceRequestContext.currentOrNull()).isSameAs(ctx);
+            try (SafeCloseable unused1 = clientRequestContext().push()) {
+                assertThat(onEnterExitStack).hasSize(2);
+                assertThat(ServiceRequestContext.currentOrNull()).isSameAs(ctx);
```

Comment:
I think we can add readability and some documentation by asserting `ClientRequestContext.current` and `RequestContext.current` too.

Reply:
That's a good suggestion. Let me add it.